lib/orwell/github/blog.ex

```diff
@@ -28,6 +28,19 @@ defmodule Orwell.GitHub.Blog do
   def posts_dir,
     do: Application.get_env(:orwell, :github_posts_dir)
 
+  @spec post_id(String.t) :: String.t
+  def post_id(path) do
+    case Regex.named_captures(~r|/(?<id>[0-9]+)[^/]*$|, path) do
+      %{"id" => id} ->
+        id
+        |> Integer.parse
+        |> elem(0)
+      _ ->
+        IO.puts "Well, fuck me dead! I tried to figure out the post id for \"#{path}\", but wasn't able to. Did the devs mess up my RegEx? I blame @mendes! BLOOP BLEEP"
```

Comment:
I was going for the slack username, but oh well (or should I say, or-well :clown_face: )